app/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ButtonGroup({ buttons, }) {
+  const style = {
+    height: `${100 / buttons.length}%`,
+  };
+
+  return (
+    <div className="buttonGroup">
+      {
+        buttons.map((button, index) => {
+          const {
+            text,
+            classes,
+            disabled,
+            onClick,
+          } = button;
```

Comment:
You can destructure right in the argument

Reply:
done